test/kitchen/kitchen-azure.yml

```diff
@@ -71,20 +71,15 @@ platforms:
     )
 
     sizes = [
-      "Standard_D1",
-      "Standard_A1",
+      "Standard_D1_v2",
+      "Standard_A1_v2",
     ]
 
     windows_sizes = [
-      "Standard_D1"
+      "Standard_D1_v2"
     ]
 
-    # we wanna spread out our quotas
-    locations = [
-      "North Central US",
-      "South Central US",
-      "Central US",
-    ]
+    location = "North Central US"
```

Comment:
Sanity check: we're sure we are fine with quotas?

Reply:
Yes